alcs-api/src/providers/typeorm/migrations/1658442355602-create_trigger_helper_functions.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class createTriggerHelperFunctions1658442355602
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE OR REPLACE FUNCTION trigger_set_timestamp()
+    RETURNS TRIGGER AS $$
+    BEGIN
+      NEW.audit_updated_at = NOW();
+      RETURN NEW;
+    END;
+    $$ LANGUAGE plpgsql;`);
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
If you remove the queryRunner from the arguments here you might be able to delete this line?